guide/src/migration.md

```diff
@@ -15,7 +15,7 @@ To migrate, switch all type casts to use `obj.downcast()` instead of `try_from(o
 
 Before:
 
-```rust
+```rust,compile_fail
```

Comment:
I think adding `# #[allow(deprecated)]` would be nicer.

Reply:
Sure thing, done 👍